### PR TITLE
feat(INJI-334): add testID for VC download flow

### PR DESCRIPTION
**Pull request by PoojaBabusing**
This PR is for issue: https://mosip.atlassian.net/browse/INJI-334

### CODE REVIEW

File: components/ui/Text.tsx [hunk 1]
```diff
@@ -16,7 +17,7 @@ export const Text: React.FC<TextProps> = (props: TextProps) => {
   ];
 
   return (
-    <RNText style={textStyles} numberOfLines={props.numLines}>
+    <RNText style={textStyles} {...testID(props.testID)}>
```
Comment (swatigoel) [inline components/ui/Text.tsx:20]: numberOfLines prop got removed. Please correct it

**Review by swatigoel — CHANGES_REQUESTED** [review]

File: components/ui/Text.tsx (hunk 1 above)
Comment (PoojaBabusing) [reply components/ui/Text.tsx]: resolved

File: components/PinInput.tsx [hunk 2]
```diff
@@ -16,7 +17,7 @@ export const PinInput: React.FC<PinInputProps> = (props) => {
   }, [state]);
 
   return (
-    <Row width="100%">
+    <Row width="100%" {...testID(props.testID)}>
```
Comment (swatigoel) [inline components/PinInput.tsx:20]: here it should be testID={props.testID}

**Review by swatigoel — CHANGES_REQUESTED** [review]

File: components/PinInput.tsx (hunk 2 above)
Comment (PoojaBabusing) [reply components/PinInput.tsx]: Done

**Review by tilak-puli — APPROVED** [review]

**Pull request merged**